llpc/patch/llpcPatchDescriptorLoad.h

```diff
@@ -76,15 +76,15 @@ class PatchDescriptorLoad:
                                 llvm::Value*        pArrayOffset,
                                 llvm::Instruction*  pInsertPoint);
 
-    ResourceMappingNodeType CalcDescriptorOffsetAndSize(ResourceMappingNodeType   nodeType1,
-                                                        ResourceMappingNodeType   nodeType2,
+    ResourceNodeType CalcDescriptorOffsetAndSize(ResourceNodeType   nodeType1,
+                                                        ResourceNodeType   nodeType2,
```

Comment:
This causes some misalignment of code lines. But I think it is not a problem since we will transit to new style and the script will help us.